src/pages/Search/index.jsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable max-len */
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import ProductSection from '../../components/ProductSection';
+import Loader from '../../components/Loader';
+import { DividerSpace } from '../../components/Mix';
+import Pagination from '../../components/Pagination';
+
+const Container = styled.div`
+  margin: 0 8%;
+  padding: 0 1rem;
+  background-color: white;
+  display: flex;
+  flex: 1 0 auto;
+  flex-direction: column;
+`;
+
+const H3 = styled.h3`
+  margin: 1rem 0;
+  display: flex;
+  flex: 1 0 auto;
+`;
+// eslint-disable-next-line max-len
+const URL = 'https://wizeline-academy.cdn.prismic.io/api/v2/documents/search?ref=YZaBvBIAACgAvnOP&q=%5B%5Bat%28document.type%2C+%22product%22%29%5D%5D&pageSize=20&languageCode=en-us&page=';
```

Comment:
Is a best practice to leave this paths and constants in a shared file across the app, somthing like  endpoints.js or constants.js